_posts/en/newsletters/2021-06-23-newsletter.md

```diff
@@ -0,0 +1,161 @@
+    participants mentioned Bitcoin Core [PR #10823][bitcoin core #10823]
+    as an alternative approach---it allows any transaction to be
+    replaced, but only after the transaction had spent a certain amount
+    of time in a node mempool (originally proposed as six hours; later
+    suggested to be 72 hours).
+
+    Both Riard's email and meeting participants noted that any proposal
+    for replacing transactions that don't contain a BIP125 opt-in signal
+    requires feedback from any merchants who are currently depending on
+    BIP125 behavior.  Optech encourages any such merchants to either
+    respond to the mailing list thread or at least communicate any concerns
+    they have to Riard and other developers directly.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+{% include specials/taproot/00-bech32m.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.13.0-beta][] is a new major release that improves feerate
+  management by making [anchor outputs][topic anchor outputs] the
+  default commitment transaction format, adds support for using a pruned
+  Bitcoin full node, allows receiving and sending payments using Atomic
+  MultiPath ([AMP][topic multipath payments]), and increases LND's
+  [PSBT][topic psbt] capabilities, among many other improvements and bug
+  fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #21365][] adds the ability for the wallet to create
+  signatures for [taproot][topic taproot] spends---both keypath spends
+  using only the P2TR public key and scriptpath spends using a
+  [tapscript][topic tapscript].  The wallet can also sign for
+  taproot-spending [PSBTs][topic psbt], but only if the wallet already
+  has all the keypath or scriptpath information it needs.  The somewhat
+  related merged PR [#22156][bitcoin core #22156] only allows importing
+  that keypath and scriptpath information after taproot is active (block
+  709,632 on mainnet, but on test networks where taproot is already
+  enabled, importing may be used now).
+
+- [Bitcoin Core #22144][] randomizes the order in which peers are serviced
+  in the message handling thread. Bitcoin Core's message handling thread is
+  responsible for parsing and processing p2p messages from peers and for
```

Comment:
s/p2p/P2P|peer-to-peer/ 

to avoid writing "message handling thread" twice in a row, perhaps join the first two sentences:

"...are serviced in Bitcoin Core's message handling thread, which is responsiblefor ..."

Reply:
I've taken both suggestions. Thanks!